### PR TITLE
[Documentation] external probe targets and custom metrics, labels

### PR DESCRIPTION
**Pull request by tbuchier**
Hello,

A minor documentation update, on some details I didn't found when learning CloudProber and had to look into source code to find those answers :)

Could we had a small doc on how to locally build and test the docs with Hugo ?
I'd like to add some other docs of external probes examples / uses cases in the future is it's okay

Thanks

### CODE REVIEW

**Review by manugarg — COMMENTED** [review]
Thank you so much for this pull request, Tanguy. Much much appreciated. I've added a few comments. Please take a look.

> Could we had a small doc on how to locally build and test the docs with Hugo ?
Good point. I just run 'hugo server' in the docs directory. I'll add this info somewhere.

> I'd like to add some other docs of external probes examples / uses cases in the future is it's okay.
Of course. You're most welcome.

File: docs/content/surfacers/overview.md [hunk 1]
```diff
@@ -112,3 +112,77 @@ surfacer {
 }
 ```
 (Source: https://github.com/google/cloudprober/blob/master/surfacers/proto/config.proto)
+
+## Additional metrics
+#### Adding your own labels
```
Comment (manugarg) [inline docs/content/surfacers/overview.md:117]: Thinking more about it, I think it's better to add it as an a new page: 

`how-to/additional-labels.md` with the following header:

```
---
menu:
    main:
        parent: "How-Tos"
        weight: 26
title: "Additional Labels"
date: 2021-06-28T17:24:32-07:00
---
```

File: docs/content/surfacers/overview.md [hunk 2]
```diff
@@ -112,3 +112,77 @@ surfacer {
...
+    key: "host"
+    value: "@target.label.fqdn@"
+  }
+
+  http_probe {}
+}
+```
+
+
+#### Adding your own metrics (external probes)
```
Comment (manugarg) [inline docs/content/surfacers/overview.md:176]: It looks like it's already covered under https://cloudprober.org/how-to/external-probe/. I am not sure if we need to repeat it here.. maybe we just need to clarify it better in external probe?

Comment (tbuchier) [reply docs/content/surfacers/overview.md]: Added 2 lines to explain the label use in external probe page and put a link to that page

**Review by manugarg — APPROVED** [review]

**Pull request merged**